aries-site/src/pages/_app.js

```diff
@@ -1,17 +1,39 @@
-import React from 'react';
+import { MDXProvider } from '@mdx-js/react';
 import PropTypes from 'prop-types';
-import { ThemeMode } from '../layouts';
-/* _app.js allows for customizing Next.js's default <App> component
- * Details: https://nextjs.org/docs/advanced-features/custom-app
- *
- * The `Component` prop is the active `page`, so whenever you
- * navigate between routes, `Component` will change to the new `page`.
- */
```

Comment:
Wondering if these comments might still be useful? ... At least the `Component` == `active page` part

Reply:
You're right. Didn't mean to lose these. will add them back in.